locale/cs.json

```diff
@@ -32,6 +32,6 @@
     "nui_warning": {
         "title": "VAROVÁNÍ",
         "warned_by": "Varován od:",
-        "instruction": "Drž [MEZERNÍK] po dobu 10 sekund pro zavření této zprávy."
+        "instruction": "Drž [MEZERNÍK] po dobu 10-ti sekund pro zavření této zprávy."
```

Comment:
Should be just "10"... What the hell is "10-ti"?

Reply:
You can use 10-ti or 10 to explain the same meaning of the word. (10 prsty a 10-ti prsty is the same and both are grammatically correct) :wink

Reply:
@czsquizer No you can not. Time to learn grammar man! https://www.meriva-preklady.cz/clanek/cesky-pravopis-10lety-18lety-pridavnejmeno_65

Reply:
Mhmhmm, I got wrong :joy: sorry. Just confused it with another thing.

Reply:
Yeah I was wrong. Trsak's version is correct